app/lib/common/pages/medications/cubit.dart

```diff
@@ -13,17 +15,51 @@ class MedicationsCubit extends Cubit<MedicationsState> {
   final int _id;
 
   Future<void> loadMedications() async {
-    final requestUri = annotationServerUrl.replace(
-      path: 'api/v1/medications/$_id',
-    );
     emit(MedicationsState.loading());
+    final response = await sendWithRandomRequests();
+    if (response == null){
+      emit(MedicationsState.error());
+    }
+    else {
+      final medication = medicationWithGuidelinesFromHTTPResponse(response);
+      emit(MedicationsState.loaded(medication));
+    }
+  }
+
+  Future<http.Response> sendRequest(Uri requestUri) async {
```

Comment:
This method should be removed completely so we can ensure that no request is sent without obfuscation